data/tlg0551/tlg017/tlg0551.tlg017.perseus-grc2.xml

```diff
@@ -174,7 +174,7 @@ fix the XML files so they are valid, make the SGML obsolete so no one is confuse
 </p>
                </div>
                <div n="16" type="textpart" subtype="section">
-                  <p>γιγνομένων δὲ τούτων ἡ βουλὴ συνῆλθεν εἰς τὸ τῆς Πίστεως ἱερόν. καί μοι θαῦμα καταφαίνεται τὸ πολλάκις ἐν τοιοῖσδε φόβοις διὰ τῆς αὐτοκράτορος ἀρχῆς διασεσωσμένους τότε μηδʼ ἐπὶ νοῦν τὸν δικτάτορα λαβεῖν, ἀλλὰ χρησιμώτατον τοῖς προτέροις τόδε τὸ ἔργον εὑρεθὲν μηδʼ ἐν μνήμῃ τοῖς πολλοῖς ἄρα γενέσθαι μήτε τότε μήθʼ ὕστερον. κρίναντες δʼ ὅσα ἔκριναν ἐς τὸ Καπιτώ̄λιον ἀνῄεσαν. καὶ πρῶτος αὐτοῖς ὁ μέγιστος ἀρχιερεὺς λεγόμενος ἐξῆρχε τῆς ὁδοῦ, Κορνήλιος Σκιπίων ὁ Νασικᾶς· ἐβόα τε μέγιστον ἕπεσθαί οἱ τοὺς ἐθέλοντας σῴζεσθαι τὴν πατρίδα καὶ τὸ κράσπεδον τοῦ ἱματίου ἐς τὴν κεφαλὴν περιεσύρατο, εἴτε τῷ παρασήμῳ τοῦ σχήματος πλέονάς οἱ συντρέχειν ἐπισπώμενος, εἴτε πολέμου τι σύμβολον τοῖς ὁρῶσιν ὡς κόρυθα ποιούμενος, εἴτε θεοὺς ἐγκαλυπτόμενος ὧν ἔμελλε δράσειν. ἀνελθόντι δὲ ἐς τὸ ἱερὸν καὶ τοῖς Γρακχείοις ἐπιδραμόντι εἶξαν μὲν ὡς κατʼ ἀξίωσιν ἀνδρὶ ἀρίστῳ, καὶ τὴν βουλὴν ἅμα οἱ θεωροῦντες ἐπιοῦσαν· οἱ δὲ τὰ ξύλα τῶν Γρακχείων αὐτῶν περισπάσαντες, ὅσα τε βάθρα καὶ ἄλλη παρασκευὴ ὡς ἐς ἐκκλησίαν συνενήνεκτο διελόντες, ἔπαιον αὐτοὺς καὶ ἐδίωκον καὶ ἐς τὰ ἀπόκρημνα κατερρίπτουν. κἀν τῷδε τῷ κυδοιμῷ πολλοί τε τῶν Γρακχείων καὶ Γράκχος αὐτός, εἱλούμενος περὶ τὸ ἱερόν, ἀνῃρέθη κατὰ τὰς θύρας παρὰ τοὺς τῶν βασιλέων ἀνδριάντας. καὶ πάντας αὐτοὺς νυκτὸς ἐξέρριψαν εἰς τὸ ῥεῦμα τοῦ ποταμοῦ.
+                  <p>γιγνομένων δὲ τούτων ἡ βουλὴ συνῆλθεν εἰς τὸ τῆς Πίστεως ἱερόν. καί μοι θαῦμα καταφαίνεται τὸ πολλάκις ἐν τοιοῖσδε φόβοις διὰ τῆς αὐτοκράτορος ἀρχῆς διασεσωσμένους τότε μηδʼ ἐπὶ νοῦν τὸν δικτάτορα λαβεῖν, ἀλλὰ χρησιμώτατον τοῖς προτέροις τόδε τὸ ἔργον εὑρεθὲν μηδʼ ἐν μνήμῃ τοῖς πολλοῖς ἄρα γενέσθαι μήτε τότε μήθʼ ὕστερον. κρίναντες δʼ ὅσα ἔκριναν ἐς τὸ Καπιτώλιον ἀνῄεσαν. καὶ πρῶτος αὐτοῖς ὁ μέγιστος ἀρχιερεὺς λεγόμενος ἐξῆρχε τῆς ὁδοῦ, Κορνήλιος Σκιπίων ὁ Νασικᾶς· ἐβόα τε μέγιστον ἕπεσθαί οἱ τοὺς ἐθέλοντας σῴζεσθαι τὴν πατρίδα καὶ τὸ κράσπεδον τοῦ ἱματίου ἐς τὴν κεφαλὴν περιεσύρατο, εἴτε τῷ παρασήμῳ τοῦ σχήματος πλέονάς οἱ συντρέχειν ἐπισπώμενος, εἴτε πολέμου τι σύμβολον τοῖς ὁρῶσιν ὡς κόρυθα ποιούμενος, εἴτε θεοὺς ἐγκαλυπτόμενος ὧν ἔμελλε δράσειν. ἀνελθόντι δὲ ἐς τὸ ἱερὸν καὶ τοῖς Γρακχείοις ἐπιδραμόντι εἶξαν μὲν ὡς κατʼ ἀξίωσιν ἀνδρὶ ἀρίστῳ, καὶ τὴν βουλὴν ἅμα οἱ θεωροῦντες ἐπιοῦσαν· οἱ δὲ τὰ ξύλα τῶν Γρακχείων αὐτῶν περισπάσαντες, ὅσα τε βάθρα καὶ ἄλλη παρασκευὴ ὡς ἐς ἐκκλησίαν συνενήνεκτο διελόντες, ἔπαιον αὐτοὺς καὶ ἐδίωκον καὶ ἐς τὰ ἀπόκρημνα κατερρίπτουν. κἀν τῷδε τῷ κυδοιμῷ πολλοί τε τῶν Γρακχείων καὶ Γράκχος αὐτός, εἱλούμενος περὶ τὸ ἱερόν, ἀνῃρέθη κατὰ τὰς θύρας παρὰ τοὺς τῶν βασιλέων ἀνδριάντας. καὶ πάντας αὐτοὺς νυκτὸς ἐξέρριψαν εἰς τὸ ῥεῦμα τοῦ ποταμοῦ.
```

Comment:
Looks like a hyphen was converted to a combining macron? If so, this may be lurking elsewhere.